packages/odata-v2/src/de-serializers/converters.ts

```diff
@@ -39,7 +39,7 @@ export function serializeFromTime(value: Time): string {
  */
 export function deserializeToMoment(edmDateTime: string): moment.Moment {
   const dateTimeOffsetComponents =
-    /^\/Date\((?<ticks>\d+)((?<sign>[+-])(?<offset>\d{4}))?\)\/$/.exec(
+    /^\/Date\((?<ticks>-{0,1}\d+)((?<sign>[+-])(?<offset>\d{4}))?\)\/$/.exec(
```

Comment:
We do already have a "sign" group. How does that relate to what we're changing here?

Reply:
<img width="578" alt="Screenshot 2022-06-20 at 15 29 58" src="https://user-images.githubusercontent.com/2292245/174612530-430cd46b-86b8-4d28-9c0d-327f88097a5e.png">

Not sure if this is an issue, but I find it at least confusing that we have a negative timestamp where the sign is `+`.

Reply:
```suggestion
    /^\/Date\((?<signedticks>-{0,1}\d+)((?<offsetsign>[+-])(?<unsignedoffset>\d{4}))?\)\/$/.exec(
```
[pp] I don't mind the current implementation but maybe this improves clarity?

Reply:
That makes it more clear to me, thanks